CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [0.2.0] 2019-07-29
```

Comment:
Change to "2019-YY-ZZ" while you don't know the exact date. I think, usually Aaron updates value to the correct date

Reply:
Thank you, Pavlo
[Resolved](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/commit/cb3fe82a7bcb91f35597a7d3cd267776c6f60a44)